packages/discord-bot/src/pre-start/env-required.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
- Can we add a comment to the top of this file to note that this is where devs should specify required env variables?